cmd/util/ledger/migrations/legacy_controller_migration_test.go

```diff
@@ -23,6 +32,15 @@ func createPayloadKeyWithLegacyController(a flow.Address, key string) ledger.Key
 	}
 }
 
+func createMigratedPayloadKey(a flow.Address, key string) ledger.Key {
+	return ledger.Key{
+		KeyParts: []ledger.KeyPart{
+			ledger.NewKeyPart(0, a.Bytes()),
+			ledger.NewKeyPart(2, []byte(key)),
```

Comment:
Can we use `KeyPartOwner | KeyPartController | KeyPartKey` here?

Reply:
fixed at 6706c4d